codec-native-quic/src/main/c/netty_quic_quiche.c

```diff
@@ -719,6 +719,39 @@ static jobject netty_new_socket_address(JNIEnv* env, const struct sockaddr_stora
     return (*env)->NewObject(env, inetsocketaddress_class, inetsocketaddress_class_constructor, address, port);
 }
 
+static jobjectArray netty_quiche_conn_path_stats(JNIEnv* env, jclass clazz, jlong conn, jlong idx) {
+    quiche_path_stats stats = {0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0};
+    quiche_conn_path_stats((quiche_conn *) conn, idx, &stats);
```

Comment:
do we need to validate that the idx is valid first ?

Reply:
Well ideally we should, however that depends on the paths_count of quiche_stats and there doesn't seem like there's another way to validate it.